indico/modules/events/persons/controllers.py

```diff
@@ -152,21 +148,32 @@ def _checkParams(self, params):
     def _process(self):
         person_ids = request.form.getlist('person_id')
         recipients = {p.email for p in self._find_event_persons(person_ids) if p.email}
-        form = EmailEventPersonsForm(person_id=person_ids, recipients=', '.join(recipients))
+        if 'no_account' in request.args:
+            tpl = get_template_module('events/registration/emails/pending_submitter.html', event=self.event_new)
+            default_body = tpl.get_html_body()
+            form = EmailEventPersonsForm(person_id=person_ids, recipients=', '.join(recipients), body=default_body)
+        else:
+            form = EmailEventPersonsForm(person_id=person_ids, recipients=', '.join(recipients))
```

Comment:
no need to repeat this, just set `body = ''` before the `if` and always pass it to this constructor